dockerfiles/incr.sh

```diff
@@ -80,9 +85,24 @@ if [[ -z "$FORCE" ]] ; then
     PREV_COMMIT=$(echo "${PREV}" | sed -e "s/-${BUILD_TYPE}.*$//")
   fi
 
+# Work out which candidate branch this one was probably forked from
+
+
   # If not found above, look for latest tagged
   if [[ -z ${PREV} ]] ; then
-    PREV=$(git log --oneline --no-merges | grep "^[0-9a-f]* Split off " | head -1 | sed "s/.*Split off //" | head -1)-rc1
+    best=1000000
+    for candidate in `git branch -r -l ${UPSTREAM}/candidate-8* | sed 's/\*//' | sed s/.*candidate-// | egrep -v [.]x | sort | uniq` ; do
```

Comment:
>egrep -v [.]x

note: will be fine, but might be safer/more explicit to make it:
egrep -v '\.x$'

Reply:
question: is there any benefit from making this a reverse sort, so it would see most recent branch 1st ?

probably not, as you can't tell if another older branch is a closer match and have to through them all..

Reply:
There will be a candidate branch with * on it if one is currently checked out